packages/@sfcc-bff/productapi/src/api/schema/productSearchResolvers.js

```diff
@@ -5,75 +5,50 @@
     For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 */
 import CommerceSdk from 'commerce-sdk';
+import utilities from '../helpers/utils.js';
```

Comment:
same querstion with all the changes in this file.

Reply:
same as above